### PR TITLE
All the things

### PR DESCRIPTION
**Pull request by amiller27**
Finished implementing tasks necessary for initial AI development.

Made the sim human-playable, i.e. a person can click things to make the drone execute actions.  It's a good time.

Misc other changes and fixes.

### CODE REVIEW

**Comment by asaba96** [comment]
Holy crap this "game" is hard :( Any way to make the touch targets for clicking the rumbas bigger? I couldn't always click on it, as it wouldn't always register I clicked on it.

File: roombasim/graphics/display.py [hunk 1]
```diff
@@ -57,16 +66,46 @@ def on_draw(self):
...
+                    Display._draw_target_roomba(r,
+                                                self.environment.target_type)
+                else:
+                    Display._draw_target_roomba(r)
             else:
                 Display._draw_obstacle_roomba(r)
 
         Display._draw_drone(self.environment.agent)
 
+    def on_mouse_release(self, x, y, button, modifiers):
```
Comment (Riteshlogin) [inline roombasim/graphics/display.py:86]: Andrew said that roombas were hard to click on, maybe increase the leeway by adding a small constant to the roomba_radius.

File: roombasim/agent/drone.py [hunk 2]
```diff
@@ -83,38 +84,50 @@ def control(self, xy_accel, yaw_vel, z_vel):
         - z_vel : a value containing the target z velocity
         '''
         self.xy_accel = np.array(xy_accel, dtype=np.float64)
+        if np.linalg.norm(self.xy_accel) > cfg.DRONE_MAX_HORIZ_ACCEL:
```
Comment (Riteshlogin) [inline roombasim/agent/drone.py]: This needs a comment

**Review by Riteshlogin — COMMENTED** [review]
Ignore this

**Review by Riteshlogin — APPROVED** [review]

File: roombasim/graphics/display.py (hunk 1 above)
Comment (amiller27) [reply roombasim/graphics/display.py:86]: Fixed

File: roombasim/agent/drone.py (hunk 2 above)
Comment (amiller27) [reply roombasim/agent/drone.py]: Fixed

**Comment by hgarrereyn** [comment]
I'm a bit late to this PR but this is awesome work! Using the human controller really makes it feel alive, although it's beginning to seem like it will be difficult for an AI to focus on more than one roomba at once. My "human" strategy has been to harass one roomba until it leaves the arena which is difficult to do consistently/efficiently.

**Pull request merged**